draft-wang-ppm-differential-privacy.md

```diff
@@ -151,19 +159,96 @@ TODO Abstract
 
 # Introduction
 
-Secure multi-computation systems like Distributed Aggregration Protocol
-{{!DAP=I-D.draft-ietf-ppm-dap-04}} enables secure aggregation of a set of
-reports submitted by Clients. DAP and the cryptographic computation scheme
-(Verifiable Distributed Aggregation Function {{!VDAF=I-D.draft-irtf-cfrg-vdaf-05}})
-it uses provides privacy by prevent any server from learning Client's
-individual contributions, as long as at least one of the aggregation servers is
-honset.
-
-This privacy guarantee can be strengthened by applying differential privacy on
-the aggregation shares output by DAP. This document describes notions and
-mechanisms of differential privacy and the methods to apply them in a DAP/VDAF
-system.
-
+Multi-party computation systems like the Distributed Aggregration Protocol
+{{!DAP=I-D.draft-ietf-ppm-dap-05}} enable secure aggregation of measurements
+generated by individuals without handling the measurements in the clear. This
+is made possible by using a Verifiable Distributed Aggregation Function
+{{!VDAF=I-D.draft-irtf-cfrg-vdaf-06}}, the core cryptographic component of DAp.
+Execution of A VDAF involves: a large set of "Clients" who produce
+cryptographically protected measurements, called "reports"; a small number of
+"Aggregators" who consume reports and produce the cryptographically protected
+aggregate; and a "Collector" who consumes the aggregate result. Distributing
+the computation of the aggregate in this manner ensures that, as long as one
+Aggregator is honest, no attacker can learn an honest Client's measurement.
+
+Depending on the application, protecting the measurements may not be sufficient
+for privacy, since the aggregate itself can reveal privacy-sensitive
+information. As an illustrative example, consider using DAP/VDAF to summarize
+the distribution of the heights of respondents to a survey. If one of the
+respondents is especially short or tall, then their contribution is likely to
+skew the summary statistic in a way that reveals their height. Ideally, no
+individual measurement would have such a signficant impact on the aggregate
+result, but in general such leakage is inevitable.
+
+This intuition can be formalized by the notion differential privacy {{Dwo06}}.
```

Comment:
Is this a good reference or would you guys prefer something else? I don't have a strong opinion here, I just picked this one because it originally formalized DP.